### ISSUE TITLE
Combined View of Threads and Microblogs

### ISSUE THREAD

**Issue by BentiGorlich** [MbinOrg/mbin#484]
Blocked by #453

**Comment by aaronliu0130** [comment]
[Relevant kbin commit](https://codeberg.org/Kbin/kbin-core/commit/2bfc8ac89d3819d186acf5922010ac90d18c57be)

**Comment by BentiGorlich** [comment]
Kbin commits are just not relevant anymore :sweat_smile: we diverged so much in so many places that we just need to find our own way. 
I have a pretty good idea how to implement this, I just need more time, as I am tackling more stuff in the background so our compatibility with other software  (mainly lemmy) increases.

**Comment by melroy89** [comment]
Rationale: This ticket apparently needs to be resolved in order to also support the new Interstellar timeline.

**Comment by jwr1** [comment]
> Rationale: This ticket apparently needs to be resolved in order to also support the new [Interstellar](https://github.com/jwr1/interstellar?rgh-link-date=2025-03-25T13%3A44%3A26.000Z) timeline.

No, you misunderstood; the timeline view works as is right now. That's by making separate API requests for both thread and microblog posts, and then it manually combines them on the client side.

I only mentioned this issue because I was saying it would be even better if it could be done server-side and officially supported by Mbin. But, the new timeline view still works with Interstellar even without official Mbin support.

**Comment by melroy89** [comment]
Ah I see. So still a nice to have.
